figures.py

```diff
@@ -0,0 +1,21 @@
+import matplotlib.pyplot as plt
+import os
+from constants import *
+import mne
+from pathlib import Path:wq
+
+
+
+
+
+def psd_plots(rec_folder_name):
+    folder_path = os.path.join(RECORDINGS_DIR, rec_folder_name, 'raw.fif')
+    raw = mne.io.read_raw_fif(folder_path)
+    folder_path_fig = os.path.join(RECORDINGS_DIR, rec_folder_name, "figures")
```

Comment:
save `os.path.join(RECORDINGS_DIR, rec_folder_name)` in variable `rec_folder_path`